apps/frontend/src/shared-research/api.ts

```diff
@@ -35,7 +35,8 @@ export const researchOutputTypeFilters: Record<
   ResearchOutputType,
   { filter: string }
 > = {
-  Proposal: { filter: 'type:Proposal' },
+  'Grant Document': { filter: 'type:Proposal OR type:"Grant Document"' },
+  Proposal: { filter: 'type:Proposal OR type:"Grant Document"' },
```

Comment:
so you are leaving Proposal in the api yeah?

Reply:
That's true